lms/templates/admin/instances.html.jinja2

```diff
@@ -7,6 +7,12 @@ Application instances
 
 {% block content %}
 
+<div class="block has-text-right">
+    <a class="button is-primary" href="{{request.route_url("admin.instance.new")}}">
+        New LTI 1.1 application instance
+    </a>
+</div>
```

Comment:
Is this OTT or is it good to be explicit? The 1.3 journey from the moment is from the registration first. Maybe one day we can have a nicer single journey?

Reply:
Makes sense to be explicit, this is a bit confusing, AI, registrations, LTI versions and orgs...

Alternatively we could just take `deployment_id` in a single journey, maybe simple to understand at the expense to have to copy the deployment_id around. 

Probably something we can leave for later, sprinkle some JS and have a nice "find and select" component for getting the deployment.

Reply:
Maybe even the button for LTI 1.3 that now says "Add instance" could change to say "Add LTI 1.3 application instance".

It clear after doing it a few times but text is cheap.

Reply:
I was thinking we might want to split the "upgrade an instance" and "create a new instance" and have them as different buttons.